db/models.py

```diff
@@ -1 +1,54 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(
+        max_length=255,
+        unique=True
+    )
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(
+        max_length=255,
+        unique=True
+    )
+    bonus = models.CharField(
+        max_length=255,
+    )
+    race = models.ForeignKey(
+        Race,
+        on_delete=models.PROTECT
```

Comment:
PROTECT is not the best option here. The race can be deleted and skills should be deleted as well

Reply:
You had PROTECT in the Player model and it was perfect. In that case you couldn't delete race when players exist. But when there are no players you can delete race and skills should be deleted as well. Now you changed race to CASCADE in Player model too. It's also possible depending on your logic but PROTECT there was better.

Reply:
Than

> You had PROTECT in the Player model and it was perfect. In that case you couldn't delete race when players exist. But when there are no players you can delete race and skills should be deleted as well. Now you changed race to CASCADE in Player model too. It's also possible depending on your logic but PROTECT there was better.

Thanks)